libraries/anaconda3-python27/Dockerfile

```diff
@@ -0,0 +1,4 @@
+ENV REQFILE=/home/algo/dependency.yml
+COPY --chown=0:0 anaconda3-buildtime/context/build /usr/local/bin/dependency-build
+COPY --chown=0:0 anaconda3-python27/context/requirements.yml $REQFILE
+RUN dependency-build $REQFILE && rm $REQFILE
```

Comment:
This is an example of how one would install an anaconda dependency, the specific operation requested is described in the `requirements.yml` file - the rest is boilerplate and reuses the `build` file defined in `anaconda3-buidltime`

Reply:
why do we remove the file? Not sure it is really necessary. I feel like it could help with debugging and it's such a small file that it won't matter from the space perspective